examples/service-clients/azure-client/external-controller/package.json

```diff
@@ -40,6 +40,7 @@
 	},
 	"dependencies": {
 		"@fluidframework/azure-client": "workspace:~",
+		"@fluidframework/map": "workspace:~",
```

Comment:
Note: Previously this example imported 'SharedMap' via the aggregate 'fluid-framework' package.  (See change description for more details.)